honeybee_radiance/cli/view.py

```diff
@@ -191,7 +195,12 @@ def merge_view(input_folder, base_name, extension, scale_factor, folder, name):
 
         # search for a single .vf in the folder and, if it's found, grab the info
         views = sorted(f for f in os.listdir(input_folder) if f.endswith('.vf'))
-        if len(views) == 1:  # replace the header with the info in the view
+        if view:
+            view_obj = View.from_file(view)
+            getinfo = Getinfo(output=output_file)
+            getinfo.options.a = 'VIEW= {}'.format(view_obj)
+            pfilt.pipe_to = getinfo
```

Comment:
Really nice! I imagine you have tested this already but It will be good to add a test for this functionality.

Reply:
Yes, I ran the command with the new view option and it appended correctly to the header.